.github/workflows/imagescan.yaml

```diff
@@ -0,0 +1,26 @@
+name: Security checks
+on: [push]
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+      with:
+        fetch-depth: 0
+    - name: Build the Docker image
+      run: make bin/submariner-engine package/.image.submariner
+    - uses: anchore/scan-action@v1
+      with:
+        image-reference: "quay.io/submariner/submariner:devel"
+        dockerfile-path: "package/Dockerfile.submariner"
+        fail-build: true
+        acs-report-enable: true
+        #acs-report-severity-cutoff: "Medium"
+    - name: anchore inline scan JSON results
```

Comment:
Maybe we can capitalize the first letters of these names, as proposed in https://github.com/submariner-io/submariner/pull/782.